src/proj_types.jl

```diff
@@ -0,0 +1,50 @@
+## Types
+
+# Projection context.  TODO: Will this be exposed?
+#type Context
+#    rep::Ptr{Void} # Pointer to internal projCtx struct
+#end
+
+@doc """
+Cartographic projection type
+""" ->
+type Projection
+    #ctx::Context   # Projection context object
+    rep::Ptr{Void} # Pointer to internal projPJ struct
+    
+    # [geod]: a structure containing the parameters of the spheroid
+    # some of the fields in [geod] are mildly duplicative of the information
+    # available in [rep], which can be exposed only through _get_spheroid_defn
+
+    # It's needed as an argument for computing great circle distances though,
+    # and for most applications, users will only have to deal with a small number
+    # of projection objects, and so we precompute it for each Projection
+    geod::geod_geodesic
```

Comment:
Or should I say, always initialize it.  May be insignificant in practice I suppose.